tests/avocado/parameters.yaml

```diff
@@ -0,0 +1,62 @@
+architecture: !mux
+    # Set the architecture of the qemu binary to execute tests
+    # with. This setting has no effect if you're using custom
+    # qemu_bin configuration.
+    x86_64:
+        arch: x86_64
```

Comment:
As I mentioned previously, I'd uncomment everything and without params I'd supply sane default so people who don't run it with `parameters.yaml` would get host system and those who use it would get all variants (unless they use `--mux-filter-only` which should be advertised).